agrest/src/main/java/io/agrest/meta/AgEntity.java

```diff
@@ -22,4 +22,6 @@
 	Collection<AgRelationship> getRelationships();
 
 	AgRelationship getRelationship(String name);
+
+	AgRelationship getRelationship(AgEntity entity);
```

Comment:
I don't think we need this. All this logic should probably happen at the Cayenne level. Also the method is ambiguous . There can be more than one relationship between two entities.